src/styles/components/DataGridWithInfinteScroll.scss

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+@import "../common/colors";
+@import "../common/fonts";
+@import "../common/borders";
+
+$csg-row-height: 48px;
+$csg-header-height: 48px;
+$table-border: 1px solid $gray-300;
+
+.data-grid-inifnite-table {
+  width: 100%;
+  flex: 1;
+  border: $table-border;
+
+  thead {
+    border: $table-border;
+    border-bottom: unset;
+  }
+
+  tbody {
+    border: $table-border;
+
+    tr {
+      &:hover {
+        background-color: $gray-100;
+      }
+
+      &:focus {
+        background-color: $blue-50;
+      }
+
+      &:active {
+        background-color: $blue-100;
+      }
+    }
+  }
+
+  tr {
+    border-top: $table-border;
+  }
+
+  th {
+    position: relative;
+    font-size: 14px;
+    font-weight: 500;
+    color: $gray-900;
+    height: 56px;
+    line-height: 35px; // To vertically align content in case of inline block
+
+    ::after {
+      content: "";
+      position: absolute;
+      right: 0;
+      top: 25%;
+      bottom: 0;
+      color: transparent;
+      border-right: $table-border;
+      max-height: 24px;
+    }
+
+    &:last-child {
+      ::after {
+        content: "";
+        border: unset;
+      }
+    }
+  }
+
+  td,
+  th {
+    padding: 0 0 0 15px;
+  }
+
+  .header-cell {
+    display: flex;
+    align-items: center;
+    font-family: $dell-font;
+
+    .arrow-up:hover,
+    .arrow-down:hover {
+      color: $blue-600;
+    }
+  }
+
+  .csg-row {
+    height: $csg-row-height;
+    line-height: $csg-row-height;
+  }
+
+  .csg-header {
+    height: $csg-header-height;
+    line-height: $csg-header-height;
+    font-weight: 500;
+  }
+
+  .csg-row,
+  .csg-header {
+
+    td,
+    th {
+      height: 48px;
+      line-height: inherit;
+      padding-left: 0.563rem;
+      padding-right: 0.563rem;
```

Comment:
Please create constants for re-used values. Still not following the SCSS format. Please go through SCSS training

Reply:
Will create the constant for re used values, Followed one of the scss file of Dell design system.

Reply:
Done